include/dlaf_c/init.h

```diff
@@ -26,4 +28,8 @@ DLAF_EXTERN_C void dlaf_initialize(int argc_pika, const char** argv_pika, int ar
                                    const char** argv_dlaf);
 
 /// Finalize DLA-Future and pika runtime
+///
+/// @post The pika runtime is fnialized and stopped when this function returns
```

Comment:
```suggestion
/// @post The pika runtime is finalized and stopped when this function returns
```

Reply:
Thanks! I need to setup a spell-checker...